requirements.txt

```diff
@@ -6,7 +6,6 @@ dynaconf>=3.1.2,<4.0.0
 fsspec>=2021.4, <=2022.1
 gitpython~=3.0
 jmespath>=0.9.5, <1.0
-jupyter_client>=5.1, <8.0
```

Comment:
This is already in the project template requirements, which is where it really belongs.

Reply:
As noted elsewhere I've actually removed `jupyter_client` altogether:
> This is a core dependence of jupyter so I've decided to not list it explicitly so we don't have to keep bumping the version requirements.